models/event.js

```diff
@@ -18,7 +18,11 @@ const MODEL = {
         .example(123345),
     parentEventId: Joi
         .number().integer().positive()
-        .description('Identifier of the parent event')
+        .description('Identifier of the direct parent event')
+        .example(123344),
+    groupEventId: Joi
```

Comment:
shouldn't this be optional in-case we merge this PR and rest of PRs are slow to merge ?

Reply:
this should be optional unless we are going to back fill older events, also we need db migration to add this column

Reply:
I don't think you need to mark fields in the model as optional.
You indicate "optional" fields below in the exports.

Reply:
where does that happen ?

Reply:
for example, these would be required: https://github.com/screwdriver-cd/data-schema/blob/master/models/event.js#L112
these would be optional: https://github.com/screwdriver-cd/data-schema/blob/master/models/event.js#L114-L115

it's done using `mutate`: https://github.com/screwdriver-cd/data-schema/blob/master/lib/mutate.js